sandbox/rocky/tf/plotter/plotter.py

```diff
@@ -0,0 +1,74 @@
+import atexit
+from queue import Empty,Queue
+from rllab.sampler.utils import rollout
+import numpy as np
+from threading import Thread
+import tensorflow as tf
+
+__all__ = [
+    'init_worker',
+    'init_plot',
+    'update_plot',
+    'shutdown_worker'
+]
+
+process = None
```

Comment:
why is this called process if it stores a thread?

Reply:
Done